about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+          <a  class="logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+          <ul class="nav-menu">
+            <li class="nav-item">
+            <a class="nav-link" id="nav1"  href="index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+              <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+          <li class="nav-item">
+            <a class="nav-link"  href="#">News</a>
+          </li>         
+          <li class="nav-item GE">
+              <a class="nav-link" id="ge" href="#">Campaign</a>
+            </li>    
+        </ul>
+        </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. <br/>Our environment comprises all living and non-living components and their interactions within a natural habitat.<br/> Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources.<br/>Promoting sustainable production of safer foods through cleaner, more efficient and cost effective technologies for improved livelihoods and national development. </p>
+      </header>
```

Comment:
- [ ] If you notice, the components of the banner section are aligned at the same level as the logo. I kindly suggest that you consider aligning the texts further to the left so that your texts are in the same level as the logo. 
- [ ] In addition, for semantic purposes consider wrapping the banner section insider the `section tag` instead of `header` tag. Header tag is only and mostly used for the menu section. This is to ensure that you improve the SEO of the application ☺️.

Your design ✔️ 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/69d56934-a6be-4f9c-ba84-95c58baa9c90)

Original design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/1d5b2ca0-7d10-4457-9cf0-494b9116055b)